custom_components/localtuya/core/pytuya/__init__.py

```diff
@@ -1115,9 +1111,9 @@ async def exchange(self, command, dps=None, nodeID=None, delay=True, payload=Non
             return await self.exchange(command, dps, nodeID=nodeID)
         return payload
 
-    async def status(self, cid=None):
+    async def status(self, cid):
```

Comment:
note that cid is an optional param so status should able to be called without any parameters and return the parent status 

```suggestion
    async def status(self, cid=None):
```